### ISSUE TITLE
Cluster labels for KernelR and KernelTimeR

### ISSUE THREAD

**Issue by arunge** [Pandora-IsoMemo/DSSM#238]
There was a lot of discussion regarding the logic that currently defines the cluster labels, see https://github.com/Pandora-IsoMemo/DSSM/issues/54, https://github.com/Pandora-IsoMemo/DSSM/issues/54#issuecomment-1492233899

- [ ] However, the labels should be in a continuum.

Since we do not want to touch the logic of clustering itself (see https://github.com/Pandora-IsoMemo/DSSM/issues/54#issuecomment-1519098205),
the fastest solution is to 
- [ ] add a new column of cluster labels with a continuous range (order the old names and rename)
- [ ] display the new labels in the map
- [ ] add the column with the new labels to the export 

@f-lukas Feel free to start adding the new label names. Maybe it is good to shift the column with the old cluster names to the end of the export table for now. Please check this!

@isomemo we will implement this as soon as possible, and we may discuss later if the column with the current labels (that possibly do have gaps) should be removed from the export table.

**Comment by isomemo** [comment]
@arunge @f-lukas 
[Wanyika Ricardo 30 July 2024.xlsx](https://github.com/user-attachments/files/16623838/Wanyika.Ricardo.30.July.2024.xlsx)
![time ranges](https://github.com/user-attachments/assets/b4af7d5c-ed67-485d-b094-0275bb780aaa)
I have attached a figure that shows the gaps following the clustering of the also attached dataset. For instance, there is a jump from cluster 15 to 19.

**Comment by f-lukas** [comment]
@isomemo - this is now fixed in the beta and in the main app.

The reason for the non-continuous cluster ids here was not the extra step we take to find the temporal groups. Instead, it was because of how mclust works. If we run mclust with 10 clusters, it is possible that only 8 clusters appear in the output and the others remain empty. Nevertheless, this solution can perform better in terms of the BIC than calculating mclust directly with 8 clusters. If this is the case, cluster ids may remain empty.

This could be fixed with just a few lines of code. Now that I know the reason, I also think that it is better not to keep the old cluster ids in extra columns. This is what I have implemented.
